packages/census/setup.js

```diff
@@ -0,0 +1,23 @@
+// Waiting for environment variables to be initialized
+Meteor.startup(() => {
+  Config = {
+    appId: process.env.APP_ID,
+    appSecret: process.env.APP_SECRET,
+    rootUrl: process.env.ROOT_URL,
+    statsServerUrl: process.env.METEOR_PACKAGE_STATS_SERVER_URL,
```

Comment:
This doesn't ever seem to get set - shouldn't there be a default value that points back to  the activity server.